Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -313,6 +313,24 @@ public DoubleExponentialMovingAverage DEMA(Symbol symbol, int period, Resolution
         }
 
         /// <summary>
+        /// Creates a new <see cref="DetrendedPriceOscillator"/> indicator.
+        /// </summary>
+        /// <param name="symbol">The symbol whose DPO we want</param>
+        /// <param name="period">The period over which to compute the DPO</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to the Value property of BaseData (x => x.Value)</param>
+        /// <returns>A new registerd DetrendedPriceOscillator indicator for the requested symbol over the specified period</returns>
```

Comment:
typo, registerd -> registered

Honestly, this typo probably occurs many times in this file and has probably just been copied around since the inception of this file

Reply:
No, just once, my typo.
Fixed.